src/components/search-filters/filter-group.tsx

```diff
@@ -141,48 +164,64 @@ const FilterGroup: React.FunctionComponent<FilterGroupProps> = memo(
+                                                        filters={filters}
+                                                        setFilters={setFilters}
+                                                    />
+                                                )}
                                             </div>
                                         );
-                                    }
-                                    return (
-                                        <Checkbox
-                                            key={`${name} ${type} ${
-                                                isMobile ? 'mobile' : ''
-                                            }`}
-                                            name={`${name} ${type} ${
-                                                isMobile ? 'mobile' : ''
-                                            }`}
-                                            // label={`${name} (${count})`}
-                                            label={name}
-                                            onToggle={checked =>
-                                                onCheckToggle(checked, item)
-                                            }
-                                            checked={
-                                                !!filters.find(
-                                                    filter =>
-                                                        filter.type === type &&
-                                                        filter.name === name
-                                                )
-                                            }
-                                        />
-                                    );
-                                })}
-                        </div>
-                        {items.length > FILTER_STEP && (
-                            <Link
-                                onClick={() => setShowAll(!showAll)}
-                                sx={{ marginTop: 'inc30' }}
+                                    })}
+
+                                {items.length > FILTER_STEP && (
+                                    <Link
+                                        onClick={() => setShowAll(!showAll)}
+                                        sx={{ marginTop: 'inc30' }}
+                                    >
+                                        Show {showAll ? 'less' : 'more'}
+                                    </Link>
+                                )}
+                            </>
+                        )}
+
+                        {sortBy && (
+                            <RadioGroup
+                                name={title || 'RadioGroup'}
+                                onChange={setSort}
+                                defaultChecked={sortBy}
+                                customStyles={{
+                                    // TODO: Remove custom styling once "box-sizing: border-box" is set globally
```

Comment:
Adding a comment here for future reference:
Was seeing styling issues with new radio button component, figured out that it had to do with the lack of a global `box-sizing: border-box` declaration. @justinmckenzie and I discussed a solution that I'd hardcode the correct styles here and create a ticket to add this globally: https://jira.mongodb.org/browse/DEVHUB-1522.

This whole styling block should be removed once that ticket is completed.